functions/venue.js

```diff
@@ -702,6 +709,30 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
   admin.firestore().collection("venues").doc(venueId).update(updated);
 });
 
+exports.updateTables = functions.https.onCall((data, context) => {
+  checkAuth(context);
+
+  const venueRef = admin.firestore().collection("venues").doc(data.venueId);
```

Comment:
It's not merged yet. :/

Reply:
You can just copy/paste the function to the same location in this PR. Then when one or the other ends up being merged, it will just show as already being present.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/f6e11778a3edad587868e67cdb5bb100ff16473d

P.S. It's merged now.